ServerCore/ServerCore.csproj

```diff
@@ -44,6 +44,7 @@
   <ItemGroup>
     <Folder Include="Areas\Identity\Data\" />
     <Folder Include="Content\" />
+    <Folder Include="Pages\Resources\ps19\" />
```

Comment:
Should not be needed - for comparison, there is no line for sibling folders pd2019 or ph20...

Reply:
Removed. 

---
In reply to: [295581650](https://github.com/PuzzleServer/mainpuzzleserver/pull/491#discussion_r295581650) [](ancestors = 295581650)